app/views/orders/_form.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-6 order-md-2" style="border: 2px solid #000; padding: 20px;">
+      <%= form_with model: order do |form| %>
+        <h3>Add your information</h3>
+        <div>
+          <h5><%= form.label :address %></h5>
+          <%= form.text_field :address, class: "form-control" %>
+          <% order.errors.full_messages_for(:address).each do |message| %>
+            <div><%= message %></div>
+          <% end %>
+        </div>
+      
+        <div>
+          <%= form.hidden_field :total_bill, value: current_user.cart.line_items.sum(:total_price) %>
+        </div>
+        
+        <div>
+          <%= form.hidden_field :status, value: 'dispatched' %>
```

Comment:
Why are we sending this value from here ? Can we use default value for status in the database ?